lib/mayaUsd/ufe/UsdScaleUndoableCommand.cpp

```diff
@@ -17,65 +17,60 @@
 
```

Comment:
Mostly same as rotate: most of the code moved to the helper base class, and undo is now implemented.